### PR TITLE
Rename raft-ann-bench to cuvs-bench and update to use the current cuvs-bench package

### PR DESCRIPTION
**Pull request by dantegd**
PR renames the `raft-ann-bench` to `cuvs-bench` and updates the run scripts so that we use the cuVS bench package as opposed to RAFT, alongside small updates needed in the calls.

Closes https://github.com/rapidsai/docker/pull/660

### CODE REVIEW

File: .github/workflows/build-test-publish-images.yml [hunk 1]
```diff
@@ -87,35 +87,35 @@ jobs:
...
+          echo "CUVS_BENCH_CPU_TAG_PREFIX=${CUVS_BENCH_CPU_TAG_PREFIX}" | tee -a ${GITHUB_OUTPUT}
       - name: Compute image repo
         id: compute-image-repo
         run: |
           base_repo="base"
           notebooks_repo="notebooks"
-          raft_ann_bench_repo="raft-ann-bench"
-          raft_ann_bench_datasets_repo="raft-ann-bench-datasets"
-          raft_ann_bench_cpu_repo="raft-ann-bench-cpu"
+          CUVS_bench_repo="cuvs-bench"
```
Comment (bdice) [inline .github/workflows/build-test-publish-images.yml:104]: Let's match case, either all upper or all lower. This applies from here on down.
```suggestion
          cuvs_bench_repo="cuvs-bench"
```

**Review by divyegala — APPROVED** [review]

File: context/cuvs-bench/get_datasets.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# Copyright (c) 2023, NVIDIA CORPORATION.
```
Comment (bdice) [inline context/cuvs-bench/get_datasets.sh:2]:
```suggestion
# Copyright (c) 2024, NVIDIA CORPORATION.
```

File: context/cuvs-bench/get_datasets.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+set -eo pipefail
+
+export CONDA_PREFIX=/opt/conda
```
Comment (bdice) [inline context/cuvs-bench/get_datasets.sh:6]: Is this line used? If so, should we activate the appropriate conda environment rather than hardcode this value?

File: cuvs-bench/README.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
-# RAFT ANN Benchmarks docker
+# cuvs ANN Benchmarks docker
```
Comment (bdice) [inline cuvs-bench/README.md:1]:
```suggestion
# cuVS Benchmarks Docker Images
```

File: cuvs-bench/README.md [hunk 5]
```diff
@@ -1,8 +1,8 @@
...
 
-This folder contains the dockerfiles for generating GPU and CPU RAFT ANN benchmark images.
+This folder contains the dockerfiles for generating GPU and CPU cuvs ANN benchmark images.
```
Comment (bdice) [inline cuvs-bench/README.md:3]:
```suggestion
This folder contains the Dockerfiles for generating GPU and CPU cuVS benchmark images.
```

File: cuvs-bench/README.md [hunk 6]
```diff
@@ -1,8 +1,8 @@
...
 
-These images are meant to enable end users of RAFT's ANN algorithms to easily run and reproduce benchmarks and comparisons between RAFT and third party libraries.
+These images are meant to enable end users of cuvs's ANN algorithms to easily run and reproduce benchmarks and comparisons between cuvs and third party libraries.
```
Comment (bdice) [inline cuvs-bench/README.md:5]: Capitalize "cuVS" throughout.
```suggestion
These images are meant to enable end users of cuVS ANN algorithms to easily run and reproduce benchmarks and comparisons between cuVS and third party libraries.
```

File: cuvs-bench/README.md [hunk 7]
```diff
@@ -13,15 +13,15 @@ There are two image types:
 
 # Running the Containers
 
-For complete details, refer to RAFT's documentation https://docs.rapids.ai/api/raft/nightly/raft_ann_benchmarks/#installing-the-benchmarks.
+For complete details, refer to cuVS's documentation https://docs.rapids.ai/api/cuvs/nightly/cuvs_bench/#installing-the-benchmarks.
```
Comment (bdice) [inline cuvs-bench/README.md:16]: It's slightly less awkward if we avoid the possessive.
```suggestion
For complete details, refer to the cuVS documentation: https://docs.rapids.ai/api/cuvs/nightly/cuvs_bench/#installing-the-benchmarks
```

File: cuvs-bench/README.md [hunk 8]
```diff
@@ -13,15 +13,15 @@ There are two image types:
...
+- `cuvs-bench`: Contains GPU and CPU benchmarks, can run all algorithms supported. Will download million-scale datasets as required. Best suited for users that prefer a smaller container size for GPU based systems. Requires the NVIDIA Container Toolkit to run GPU algorithms, can run CPU algorithms without it.
+- `cuvs-bench-datasets`: Contains the GPU and CPU benchmarks with million-scale datasets already included in the container. Best suited for users that want to run multiple million scale datasets already included in the image.
+- `cuvs-bench-cpu`: Contains only CPU benchmarks with minimal size. Best suited for users that want the smallest containers to reproduce benchmarks on systems without a GPU.
 
-Nightly images are located in [dockerhub](https://hub.docker.com/r/rapidsai/raft-ann-bench), meanwhile release (stable) versions are located in [NGC](https://hub.docker.com/r/rapidsai/raft-ann-bench), starting with release 23.12.
+Nightly images are located in [dockerhub](https://hub.docker.com/r/rapidsai/cuvs-bench), meanwhile release (stable) versions are located in [NGC](https://hub.docker.com/r/rapidsai/cuvs-bench), starting with release 23.12.
```
Comment (bdice) [inline cuvs-bench/README.md:24]: The note about 23.12 is now outdated. Also, this NGC link is wrong but I don't know how to fix that.
```suggestion
Nightly images are located in [DockerHub](https://hub.docker.com/r/rapidsai/cuvs-bench), meanwhile release (stable) versions are located in [NGC](https://hub.docker.com/r/rapidsai/cuvs-bench).
```

File: cuvs-bench/cpu/Dockerfile [hunk 9]
```diff
@@ -3,7 +3,7 @@
 ARG PYTHON_VER=unset
 ARG RAPIDS_VER=24.12
 
-FROM condaforge/mambaforge:23.3.1-0 AS raft-ann-bench-cpu
+FROM condaforge/mambaforge:23.3.1-0 AS cuvs-bench-cpu
```
Comment (bdice) [inline cuvs-bench/cpu/Dockerfile:6]: This base image is out of date. We need to use miniforge instead of mambaforge, and we should use the latest version.
```suggestion
FROM condaforge/miniforge3:24.7.1-2 AS cuvs-bench-cpu
```

Comment (bdice) [reply cuvs-bench/cpu/Dockerfile:6]: It looks like this has stalled in another PR (https://github.com/rapidsai/docker/pull/660/) but that PR was trying to switch to micromamba rather than miniforge3. My suggestion here should be safe, afaik.

File: cuvs-bench/gpu/Dockerfile [hunk 10]
```diff
@@ -29,7 +29,7 @@ RUN apt-get install perl -y
 RUN <<EOF
 mamba update --all -y -n base
 mamba install -y -n base \
-    "raft-ann-bench=${RAPIDS_VER}.*" \
+    "cuvs_bench=${RAPIDS_VER}.*" \
```
Comment (bdice) [inline cuvs-bench/gpu/Dockerfile:32]: I wish we had named this package `cuvs-bench` instead of `cuvs_bench`... that would be more favorable to align with how most packages are named in RAPIDS (and the conda ecosystem generally).

Comment (dantegd) [reply cuvs-bench/gpu/Dockerfile:32]: I wish so too :( 

@divyegala @cjnolet  I wonder if this would be something we should try to change, since we have had only one release with the `_` named packagE?

Comment (cjnolet) [reply cuvs-bench/gpu/Dockerfile:32]: I 100% agree with this- cuvs-bench seems more natural to me (and matches  the code examles in our docs.. )

Comment (divyegala) [reply cuvs-bench/gpu/Dockerfile:32]: Yes that's totally fine. Keep in mind that the actual Python package still must be called `cuvs_bench`, even if we call the conda/pip packages `cuvs-bench`. Here's the historical reason why: https://github.com/rapidsai/raft/pull/2333

TL;DR A python module called `cuvs-bench` is not a valid Python identifier

Comment (bdice) [reply cuvs-bench/gpu/Dockerfile:32]: Yes, that's correct. We use dashes for package names and underscores for import names. e.g. `conda install dask-cudf`; `import dask_cudf`.

Comment (bdice) [reply cuvs-bench/gpu/Dockerfile:32]: I attempted to fix this here: https://github.com/rapidsai/cuvs/pull/417

File: .github/workflows/build-test-publish-images.yml [hunk 11]
```diff
@@ -87,35 +87,35 @@ jobs:
...
       - name: Compute image repo
         id: compute-image-repo
         run: |
           base_repo="base"
           notebooks_repo="notebooks"
-          raft_ann_bench_repo="raft-ann-bench"
-          raft_ann_bench_datasets_repo="raft-ann-bench-datasets"
-          raft_ann_bench_cpu_repo="raft-ann-bench-cpu"
+          cuvs_bench_repo="cuvs-bench"
+          CUVS_bench_datasets_repo="cuvs-bench-datasets"
```
Comment (bdice) [inline .github/workflows/build-test-publish-images.yml:105]: https://github.com/rapidsai/docker/pull/715/#discussion_r1805212719

> This applies from here on down.

I wasn't able to add GitHub suggestions on the entire block and didn't really want to do it line by line...

File: .github/workflows/build-test-publish-images.yml [hunk 12]
```diff
@@ -87,35 +87,35 @@ jobs:
...
+          CUVS_bench_cpu_repo="cuvs-bench-cpu"
```
Comment (bdice) [inline .github/workflows/build-test-publish-images.yml:106]:
```suggestion
          cuvs_bench_cpu_repo="cuvs-bench-cpu"
```

File: .github/workflows/build-test-publish-images.yml [hunk 13]
```diff
@@ -87,35 +87,35 @@ jobs:
...
           if [ "${{ inputs.build_type }}" = "pull-request" ]; then
             base_repo="staging"
             notebooks_repo="staging"
-            raft_ann_bench_repo="staging"
-            raft_ann_bench_datasets_repo="staging"
-            raft_ann_bench_cpu_repo="staging"
+            CUVS_bench_repo="staging"
```
Comment (bdice) [inline .github/workflows/build-test-publish-images.yml:110]:
```suggestion
            cuvs_bench_repo="staging"
```

File: .github/workflows/build-test-publish-images.yml [hunk 14]
```diff
@@ -87,35 +87,35 @@ jobs:
...
+            CUVS_bench_datasets_repo="staging"
```
Comment (bdice) [inline .github/workflows/build-test-publish-images.yml:111]:
```suggestion
            cuvs_bench_datasets_repo="staging"
```

File: .github/workflows/build-test-publish-images.yml [hunk 15]
```diff
@@ -87,35 +87,35 @@ jobs:
...
+            CUVS_bench_cpu_repo="staging"
```
Comment (bdice) [inline .github/workflows/build-test-publish-images.yml:112]:
```suggestion
            cuvs_bench_cpu_repo="staging"
```

File: .github/workflows/build-test-publish-images.yml [hunk 16]
```diff
@@ -87,35 +87,35 @@ jobs:
...
           fi
           echo "BASE_IMAGE_REPO=${base_repo}" | tee -a ${GITHUB_OUTPUT}
           echo "NOTEBOOKS_IMAGE_REPO=${notebooks_repo}" | tee -a ${GITHUB_OUTPUT}
-          echo "RAFT_ANN_BENCH_IMAGE_REPO=${raft_ann_bench_repo}" | tee -a ${GITHUB_OUTPUT}
-          echo "RAFT_ANN_BENCH_DATASETS_IMAGE_REPO=${raft_ann_bench_datasets_repo}" | tee -a ${GITHUB_OUTPUT}
-          echo "RAFT_ANN_BENCH_CPU_IMAGE_REPO=${raft_ann_bench_cpu_repo}" | tee -a ${GITHUB_OUTPUT}
+          echo "CUVS_BENCH_IMAGE_REPO=${CUVS_bench_repo}" | tee -a ${GITHUB_OUTPUT}
```
Comment (bdice) [inline .github/workflows/build-test-publish-images.yml:116]:
```suggestion
          echo "CUVS_BENCH_IMAGE_REPO=${cuvs_bench_repo}" | tee -a ${GITHUB_OUTPUT}
```

File: .github/workflows/build-test-publish-images.yml [hunk 17]
```diff
@@ -87,35 +87,35 @@ jobs:
...
+          echo "CUVS_BENCH_DATASETS_IMAGE_REPO=${CUVS_bench_datasets_repo}" | tee -a ${GITHUB_OUTPUT}
```
Comment (bdice) [inline .github/workflows/build-test-publish-images.yml:117]:
```suggestion
          echo "CUVS_BENCH_DATASETS_IMAGE_REPO=${cuvs_bench_datasets_repo}" | tee -a ${GITHUB_OUTPUT}
```

File: .github/workflows/build-test-publish-images.yml [hunk 18]
```diff
@@ -87,35 +87,35 @@ jobs:
...
+          echo "CUVS_BENCH_CPU_IMAGE_REPO=${CUVS_bench_cpu_repo}" | tee -a ${GITHUB_OUTPUT}
```
Comment (bdice) [inline .github/workflows/build-test-publish-images.yml:118]:
```suggestion
          echo "CUVS_BENCH_CPU_IMAGE_REPO=${cuvs_bench_cpu_repo}" | tee -a ${GITHUB_OUTPUT}
```

File: cuvs-bench/README.md [hunk 19]
```diff
@@ -0,0 +1,90 @@
...
+
+For complete details, refer to the cuVS documentation: https://docs.rapids.ai/api/cuvs/nightly/cuvs_bench/#installing-the-benchmarks
+
+We provide images for GPU enabled systems, as well as systems without a GPU. The following images are available:
+
+- `cuvs-bench`: Contains GPU and CPU benchmarks, can run all algorithms supported. Will download million-scale datasets as required. Best suited for users that prefer a smaller container size for GPU based systems. Requires the NVIDIA Container Toolkit to run GPU algorithms, can run CPU algorithms without it.
+- `cuvs-bench-datasets`: Contains the GPU and CPU benchmarks with million-scale datasets already included in the container. Best suited for users that want to run multiple million scale datasets already included in the image.
+- `cuvs-bench-cpu`: Contains only CPU benchmarks with minimal size. Best suited for users that want the smallest containers to reproduce benchmarks on systems without a GPU.
+
+Nightly images are located in [DockerHub](https://hub.docker.com/r/rapidsai/cuvs-bench), meanwhile release (stable) versions are located in [NGC](https://hub.docker.com/r/rapidsai/cuvs-bench).
```
Comment (bdice) [inline cuvs-bench/README.md:24]: @dantegd You marked my previous conversation as resolved but didn't answer the question -- what do we do with NGC here? This link is pointing to DockerHub. I observe a similar issue in https://github.com/rapidsai/cuvs/blob/f708fe388ee206105fd7894388283995e88ab7f9/docs/source/cuvs_bench/index.rst?plain=1#L96

File: cuvs-bench/README.md [hunk 20]
```diff
@@ -0,0 +1,90 @@
...
+    "--algorithms cuvs_cagra" \
+    ""
+```
+
+Where:
+
+```bash
+export DATA_FOLDER=path/to/store/results/and/data # <- Results and datasets will be written to this host folder.
+docker run --gpus all --rm -it \
+    -v $DATA_FOLDER:/home/rapids/benchmarks  \ # <- local folder to store datasets and results
```
Comment (bdice) [inline cuvs-bench/README.md:51]: This script is not usable. The continuation line `\` breaks the comment. https://stackoverflow.com/questions/9522631/how-to-put-a-line-comment-for-a-multi-line-command

You might want to just have comments that are separate from the code? Something like:


------

```bash
export DATA_FOLDER=path/to/store/results/and/data
docker run --gpus all --rm -it \
    -v $DATA_FOLDER:/home/rapids/benchmarks  \
    -u $(id -u) \
    rapidsai/cuvs-bench:24.12a-cuda12.5-py3.12 \
    "--dataset deep-image-96-angular" \
    "--normalize" \
    "--algorithms cuvs_cagra" \
    ""
```

where

- `$DATA_FOLDER` defines where to store datasets and results
- `-u $(id -u)` allows the container to use the host user for permissions
- `rapidsai/cuvs-bench` is the image to use (it can also be `rapidsai/cuvs-bench-datasets`)
- ...

File: cuvs-bench/README.md [hunk 21]
```diff
@@ -0,0 +1,90 @@
...
+    -u $(id -u) \ # <- this flag allows the container to use the host user for permissions
+    rapidsai/cuvs-bench:24.12a-cuda12.5-py3.12 \ # <- image to use, either `cuvs-bench` or `cuvs-bench-datasets`
+    "--dataset deep-image-96-angular" \ # <- dataset name
+    "--normalize" \ # <- whether to normalize the dataset, leave string empty ("") to not normalize.
+    "--algorithms cuvs_cagra" \ # <- what algorithm(s) to use as a ; separated list, as well as any other argument to pass to `cuvs_bench.run`
+    "" # optional argumetns to pass to `cuvs_benchmarks.plot`
```
Comment (bdice) [inline cuvs-bench/README.md:57]:
```suggestion
    "" # optional arguments to pass to `cuvs_benchmarks.plot`
```

File: cuvs-bench/README.md [hunk 22]
```diff
@@ -0,0 +1,90 @@
...
+```
+
+For CPU systems the same interface applies, except for not needing the gpus argument and using the cpu images:
+```bash
+export DATA_FOLDER=path/to/store/results/and/data
+docker run  all --rm -it \
+    -v $DATA_FOLDER:/home/rapids/benchmarks  \
+    -u $(id -u) \ # <- this flag allows the container to use the host user for permissions
```
Comment (bdice) [inline cuvs-bench/README.md:65]: This comment breaks the line continuation. See above. This must be removed for users to be able to copy-paste this command.
```suggestion
    -u $(id -u) \
```

File: cuvs-bench/README.md [hunk 23]
```diff
@@ -0,0 +1,90 @@
...
+     "--algorithms cuvs_cagra" \
+     ""
+```
+
+2. **Using the preinstalled `cuvs_bench` python package**: The docker containers are built using the conda packages described in the following section, so they can be used directly as if they were installed manually following the instructions in the next section. This allows using the full flexibility of the scripts. To use the python scripts directly, an easy way is to use the following command:
+
+```bash
+export DATA_FOLDER=path/to/store/results/and/data
+docker run --gpus all --rm -it \
+    -v $DATA_FOLDER:/home/rapids/benchmarks  \
```
Comment (bdice) [inline cuvs-bench/README.md:78]: Remove extraneous space.
```suggestion
    -v $DATA_FOLDER:/home/rapids/benchmarks \
```

File: cuvs-bench/README.md [hunk 24]
```diff
@@ -0,0 +1,90 @@
...
+    "--normalize" \ # <- whether to normalize the dataset, leave string empty ("") to not normalize.
+    "--algorithms cuvs_cagra" \ # <- what algorithm(s) to use as a ; separated list, as well as any other argument to pass to `cuvs_bench.run`
+    "" # optional argumetns to pass to `cuvs_benchmarks.plot`
+```
+
+For CPU systems the same interface applies, except for not needing the gpus argument and using the cpu images:
+```bash
+export DATA_FOLDER=path/to/store/results/and/data
+docker run  all --rm -it \
+    -v $DATA_FOLDER:/home/rapids/benchmarks  \
```
Comment (bdice) [inline cuvs-bench/README.md:64]:
```suggestion
    -v $DATA_FOLDER:/home/rapids/benchmarks \
```

File: cuvs-bench/README.md (hunk 20 above)
Comment (bdice) [inline cuvs-bench/README.md:51]:
```suggestion
    -v $DATA_FOLDER:/home/rapids/benchmarks \ # <- local folder to store datasets and results
```

File: cuvs-bench/README.md [hunk 25]
```diff
@@ -0,0 +1,90 @@
...
+The containers can be used in two manners:
+
+1. **Quick benchmark with single `docker run`**: The docker containers already include helper scripts to be able to invoke most of the functionality of the benchmarks from docker run for a simple and easy way to run benchmarks.
+
+For GPU systems, where $DATA_FOLDER is a local folder where you want datasets stored in $DATA_FOLDER/datasets and results in $DATA_FOLDER/results:
+
+```bash
+export DATA_FOLDER=path/to/store/results/and/data
+docker run --gpus all --rm -it \
+    -v $DATA_FOLDER:/home/rapids/benchmarks  \
```
Comment (bdice) [inline cuvs-bench/README.md:37]:
```suggestion
    -v $DATA_FOLDER:/home/rapids/benchmarks \
```

File: .github/workflows/build-test-publish-images.yml (hunk 11 above)
Comment (dantegd) [reply .github/workflows/build-test-publish-images.yml:105]: Apologies, missed it after apply bulk commits resolved conversations automatically!

File: cuvs-bench/README.md (hunk 19 above)
Comment (dantegd) [reply cuvs-bench/README.md:24]: Created an issue so that we add the link when the container is published https://github.com/rapidsai/docker/issues/716, I'm not entirely sure how the link will look like.

File: cuvs-bench/cpu/Dockerfile [hunk 26]
```diff
@@ -31,7 +31,7 @@ mamba update --all -y -n base
 mamba install -y -n base "python=${PYTHON_VER}"
 mamba update --all -y -n base
 mamba install -y -n base \
-  "raft-ann-bench-cpu=${RAPIDS_VER}.*" \
+  "cuvs_bench_cpu=${RAPIDS_VER}.*" \
```
Comment (bdice) [inline cuvs-bench/cpu/Dockerfile:34]: Update from https://github.com/rapidsai/cuvs/pull/417.
```suggestion
  "cuvs-bench-cpu=${RAPIDS_VER}.*" \
```

File: cuvs-bench/gpu/Dockerfile (hunk 10 above)
Comment (bdice) [reply cuvs-bench/gpu/Dockerfile:32]: Update from https://github.com/rapidsai/cuvs/pull/417.
```suggestion
    "cuvs-bench=${RAPIDS_VER}.*" \
```

**Review by bdice — COMMENTED** [review]
A couple more updates from https://github.com/rapidsai/cuvs/pull/417.

**Review by bdice — APPROVED** [review]

**Comment by bdice** [comment]
I pushed one small fix that was missed -- now things look good. Thanks for the iterations on this!

File: cuvs-bench/README.md [hunk 27]
```diff
@@ -0,0 +1,87 @@
...
+
+Where:
+
+- `DATA_FOLDER=path/to/store/results/and/data`: Results and datasets will be written to this host folder.
+- `-u $(id -u)`: This flag allows the container to use the host user for permissions
+- `rapidsai/cuvs-bench:24.12a-cuda12.5-py3.12`: Image to use, either `cuvs-bench` or `cuvs-bench-datasets`, cuVS version, CUDA version, and Python version.
+- "--dataset deep-image-96-angular": Dataset name(s). See https://docs.rapids.ai/api/cuvs/nightly/cuvs_bench for more details.
+- "--normalize": Whether to normalize the dataset, leave string empty ("") to not normalize.
+- "--algorithms cuvs_cagra" \ # <- what algorithm(s) to use as a ; separated list, as well as any other argument to pass to `cuvs_bench.run`.
+- Last line, (""): optional argumetns to pass to `cuvs-bench.plot`, pass an empty string if no parameters to plot are needed.
```
Comment (bdice) [inline cuvs-bench/README.md:54]:
```suggestion
- Last line, (""): optional arguments to pass to `cuvs_bench.plot`, pass an empty string if no parameters to plot are needed.
```

**Comment by bdice** [comment]
Oops. Looks like one conversation was unaddressed, but I don't want to block on it. https://github.com/rapidsai/docker/pull/715#discussion_r1805533452

File: cuvs-bench/README.md [hunk 28]
```diff
@@ -0,0 +1,87 @@
...
+```
+
+Where:
+
+- `DATA_FOLDER=path/to/store/results/and/data`: Results and datasets will be written to this host folder.
+- `-u $(id -u)`: This flag allows the container to use the host user for permissions
+- `rapidsai/cuvs-bench:24.12a-cuda12.5-py3.12`: Image to use, either `cuvs-bench` or `cuvs-bench-datasets`, cuVS version, CUDA version, and Python version.
+- "--dataset deep-image-96-angular": Dataset name(s). See https://docs.rapids.ai/api/cuvs/nightly/cuvs_bench for more details.
+- "--normalize": Whether to normalize the dataset, leave string empty ("") to not normalize.
+- "--algorithms cuvs_cagra" \ # <- what algorithm(s) to use as a ; separated list, as well as any other argument to pass to `cuvs_bench.run`.
```
Comment (bdice) [inline cuvs-bench/README.md:53]: I think this fixes the last remaining issue.
```suggestion
- "--algorithms cuvs_cagra": What algorithm(s) to use as a ; separated list, as well as any other argument to pass to `cuvs_bench.run`.
```

**Review by raydouglass — APPROVED** [review]

**Comment by dantegd** [comment]
@raydouglass anything else needed here or can this be merged and containers published?

**Comment by bdice** [comment]
And after this is merged, can we stop building and/or remove raft’s benchmark packages?
